recipes-kernel/rtl8723ds-mod/rtl8723ds.bb

```diff
@@ -0,0 +1,18 @@
+SUMMARY = "Linux driver for RTL8723DS"
+DESCRIPTION = "${SUMMARY}"
+LICENSE = "GPL-2.0-only"
```

Comment:
This seems to be supported upstream, do we really need this?

Reply:
If I understood correctly only bluetooth is supported upstream, not wifi. And Armbian uses this driver too https://github.com/armbian/build/blob/7bf3c297cb4a482468590c5f20dec2dd66ab2e29/lib/functions/compilation/patch/drivers_network.sh#L423 .   But I can be mistaken here...

Reply:
It looks like you are right. That's unfortunate